compiler/src/openxla/compiler/nvgpu/Conversion/ConvertHLOToCUDNN.cpp

```diff
@@ -125,14 +130,29 @@ static LogicalResult outlineToGraph(TensorValue result,
       result.getLoc(), funcType.getResults(), results);
   rewriter.create<ReturnOp>(root->getLoc(), castOp.getResults());
 
+  // Create global handle before any other initialization.
```

Comment:
Move it to a helper function, and cache it? Otherwise we'll create a handle per graph? This is somewhat similar to lazy API creating here: https://github.com/openxla/openxla-nvgpu/blob/e802bc14ed4579c34c3a79dbf42f734bf71f5b5b/compiler/src/openxla/compiler/nvgpu/Dialect/CUDNN/Conversion/ConvertCUDNNToRuntime.cpp#L710, patterns have to capture "shared state" as `shared_ptr`, but otherwise works fine

Reply:
Somehow had thought we wanted many handles as before. Will change in follow-up